packages/epo-widget-lib/src/atomic/Blinker/Blinker.tsx

```diff
@@ -0,0 +1,87 @@
+import { FunctionComponent, useState } from "react";
+import useInterval from "@/hooks/useInterval";
+import * as Styled from "./styles";
+import { Image } from "@rubin-epo/epo-react-lib/dist/types/image";
+
+export interface BlinkerProps {
+  images: Image[];
+  activeIndex: number;
+  autoplay?: boolean;
+  loop?: boolean;
+  interval?: number;
+  blinkCallback: (activeIndex: number) => void;
+}
+
+const Blinker: FunctionComponent<BlinkerProps> = ({
+  images = [],
+  activeIndex = 0,
+  autoplay = true,
+  loop = true,
+  interval = 200,
+  blinkCallback,
+}) => {
+  const [playing, setPlaying] = useState(autoplay);
+  const canBlink = images.length > 1;
+
+  const getBlink = (direction = 0) => {
+    const nextIndex = activeIndex + direction;
+    const lastIndex = images.length - 1;
+
+    let newIndex = nextIndex;
+
+    if (nextIndex > lastIndex) {
+      newIndex = 0;
+    }
+
+    if (nextIndex < 0) {
+      newIndex = lastIndex;
+    }
```

Comment:
I'm not a huge fan of this setup for determining the index if it is out of bounds of the array, it's pulled from the old component but I feel like there should be some sort of better though out clamping method.